### PR TITLE
US-11: Acceptance Test

### PR DESCRIPTION
**Pull request by RGPosadas**
See the GIF here:
https://github.com/RGPosadas/WayFinder/wiki/Acceptance-Tests-GIFs#us-11

### CODE REVIEW

**Review by cristian-aldea — APPROVED** [review]
The GIF looks good! Unfortunately, I can't run it myself. Let's wait for another review to go through.

**Review by cindyslittleplanet — APPROVED** [review]
Ran the test on MacBook Air, the test passes

**Pull request merged**
